tools/add_triage_signature.py

```diff
@@ -2037,6 +2037,52 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("\n".join(host_messages))
 
 
+class ErrorOnCleanupInstallDevice(Signature):
+    """
+    This signature monitors errors on cleanupInstallDevice function
+    """
+
+    # This has to be maintained to be the same format as
+    # https://github.com/openshift/assisted-installer/blob/51c021f3245ef1d9e1a50a3e31dc23350cdc5d32/src/installer/installer.go#L98
+    # Remember to maintain backwards compatibility if that format ever changes - create
+    # PATTERN_NEW and try to match both.
+    LOG_PATTERN = re.compile(r'time=".+" level=[a-z]+ msg="(?P<message>failed to prepare install device.*)"')
```

Comment:
Being overly specific with the searched pattern would just lead to more likely breakage on changes, IMO just finding the original substring is good enough

Reply:
Sure, changed!